tests/components/axis/test_init.py

```diff
@@ -37,17 +37,18 @@
 
 async def test_setup_entry_fails(hass):
     """Test successful setup of entry."""
-    entry = MockConfigEntry(
-        domain=axis.DOMAIN, data={axis.CONF_MAC: "0123"}, options=True
+    config_entry = MockConfigEntry(
+        domain=axis.DOMAIN, data={axis.CONF_MAC: "0123"}, options=True, version=2
     )
+    config_entry.add_to_hass(hass)
 
     mock_device = Mock()
     mock_device.async_setup.return_value = mock_coro(False)
 
     with patch.object(axis, "AxisNetworkDevice") as mock_device_class:
         mock_device_class.return_value = mock_device
 
-        assert not await axis.async_setup_entry(hass, entry)
+        assert not await hass.config_entries.async_setup(config_entry.entry_id)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4925854)